config/config.go

```diff
@@ -230,12 +233,19 @@ func (c *Config) ArchiveReadDays() int {
 	return getIntValue("ARCHIVE_READ_DAYS", defaultArchiveReadDays)
 }
 
+// RequestTimeout returns a Duration.time represent the request timeout
+func (c *Config) RequestTimeout() time.Duration {
+	requestTimout := getIntValue("REQUEST_MAX_TIMEOUT", defaultRequestTimeout)
```

Comment:
Maybe we can name the variable `HTTP_CLIENT_TIMEOUT` to be more explicit. In the future, the HTTP server timeout could be also configurable...

Reply:
done